src/OSPSuite.Core/Extensions/StringExtensions.cs

```diff
@@ -108,5 +108,14 @@ public static string StripUnit(this string stringToStrip)
 
          return indexOfUnitStart == -1 ? stringToStrip : stringToStrip.Remove(indexOfUnitStart, stringToStrip.Length - indexOfUnitStart).TrimEnd();
       }
+
+      /// <summary>
+      /// Returns the give text within quotes. 
+      /// </summary>
+      /// <example>
+      /// InQuotes(test) => "test"
+      /// </example>
+      /// <param name="text">Text to encapsulate in quotes</param>
+      public static string InQuotes(this string text) => $"\"{text}\"";
```

Comment:
Should we check here that `text` does not contains (double) quotes? 
At least **in MoBi** user can create parameters with double quotes in name; if such a parameter will be varied in a population, CSV export will be invalid then
Just created an iissue for that: https://github.com/Open-Systems-Pharmacology/MoBi/issues/530

Reply:
"" should be forbidden from parameter names. Why is this allowed in the first place..no idea